### DISCUSSION TITLE
Latest build not downloading GApps

### DISCUSSION THREAD

**Discussion by djgrege** [LSPosed/MagiskOnWSALocal#785] [Q&A]
It seems that when running the script to download everything, GApps is skipped, and the installer doesn't give the option to install GApps now. I then tired following the tutorial on how to add MindTheGapps separately but got output:
build.sh: unrecognized option '--gapps-brand'
build.sh: unrecognized option '--gapps-variant'
ERROR: Failed to parse options, please check your input
How can I fix this?

**Comment by rabilrbl** [comment]
@djgrege after recent changes, there is no support for custom gapps brand.

For Gapps support add `--install-gapps` flag

**Comment by irishmike** [comment]
Have added the flag, build script output shows "Integrating GApps":

Integrating GApps
Loading cpio: [/tmp/wsa-build-n1ia82VxV0_/wsa/x64/Tools/initrd.img]
Add file [/lspolicy.rule] (100000)
Add file [overlay.d/gapps.rc] (100000)
Add file [overlay.d/sbin/lsp_gapps.img] (100000)
Dumping cpio: [/tmp/wsa-build-n1ia82VxV0_/wsa/x64/Tools/initrd.img]
done

The install appears to run fine and  Magisk and WSA are both fine, but zero presence of the GApps

**Comment by TempleKa** [comment]
GApps only seems to install when selecting root option. Is this how it's supposed to be?

**Reply by cjust1** [reply]
I"m also having this issue.  Is there something I should be doing to get gapps to install?

**Comment by FarYetNear** [comment]
Has anyone figure this out yet?     The --install...  flag isn't working.
